clients/java/zts/core/src/main/java/com/yahoo/athenz/zts/ZTSClient.java

```diff
@@ -1168,31 +1192,41 @@ public AccessTokenResponse getAccessToken(String domainName, List<String> roleNa
             }
         }
 
+        // if no hit then we need to look up in disk
+        try {
+            accessTokenResponse = ztsFileUtil.lookupAccessTokenFromDisk(domainName, roleNames);
+        } catch (IOException e) {
+            LOG.warn("GetAccessToken: failed to load access token from disk ", e.getMessage());
```

Comment:
this should be an error if file based access tokens is enabled and we're not able to read them from disk.